wordpress-mobile/WordPress-Android
# Show message when all QS tasks from the same category are completed

Fixes #14042

This PR introduces a snackbar message to the improved My Site screen that is shown when all Quick Start tasks from a single category are completed. If the last task from that category is completed on the My Site screen, the snackbar appears immediately. If the task is completed on another screen, the snackbar appears when the user comes back to the My Site screen.

https://user-images.githubusercontent.com/830056/107836013-538b2e00-6d7a-11eb-9f14-9fedc78014a2.mp4

### To test

1. Turn on the Improved My Site flag and restart the app.
1. Go to the My Site screen and select a site where Quick Start is active.
1. Complete all tasks under the "Customize your site screen" category, but leave for last a task that is completed on the My Site screen (`UPDATE_SITE_TITLE` or `UPLOAD_SITE_ICON`).
1. After completing the last task, notice the corresponding snackbar message for that category is shown immediately.
1. Complete all tasks under the "Grow your audience" category and make sure the corresponding snackbar message is shown after you go back to the My Site screen.

---

PR submission checklist:
